src/main.rs

```diff
@@ -43,7 +43,7 @@ fn main() {
         .logo("https://www.coredump.ch/wp-content/uploads/2016/11/logo.png")
         .url("https://www.coredump.ch/")
         .location(api::Location {
-            address: Some("Holzwiesstrasse 50, 8645 Jona, Switzerland".into()),
+            address: Some("Neue Jonastrasse 107, 8640 Rapperswil, Switzerland".into()),
```

Comment:
Actually I'm not sure if most spaces implement it that way :wink: Maybe we should make it clearer in the specs by having a `postal_address` and a `address` field.

Reply:
Changing semantics of `address` would not be reverse compatible :slightly_smiling_face: Also, regarding the address I agree that the coordinates are more useful in order to find the space.

The postal address is important to know if you want to send a letter or some other stuff to another hackerspace.

Anyways, this discussion should belong in the schema repo, not here.